contracts/OrderRFQMixin.sol

```diff
@@ -230,13 +233,13 @@ abstract contract OrderRFQMixin is EIP712, OnlyWethReceiver {
 
         // Maker => Taker
         if (order.makerAsset.get() == address(_WETH) && flagsAndAmount & _UNWRAP_WETH_FLAG != 0) {
-            _WETH.transferFrom(maker, address(this), makingAmount);
+            _WETH.safeTransferFromUniversal(maker, address(this), makingAmount, (flagsAndAmount & _USE_PERMIT2_FLAG) != 0);
```

Comment:
This should be done in taker => maker flow.

Maker => taker flow should be controlled by maker. 129 bit of order.info would be good for that.